EIPS/eip-2357.md

```diff
@@ -0,0 +1,38 @@
+---
+eip: 2357
+title: Total Difficulty in Block Header
+author: Tomasz K. Stanczak (@tkstanczak)
+discussions-to: https://ethereum-magicians.org/t/eip-2357-replace-difficulty-with-total-difficulty-in-block-headers/4233
+status: Draft
+type: Standard Track
+category: Core
+created: 2020-04-29
+---
+
+## Simple Summary
+
+This document proposes to replace Dififculty with Total Difficulty in the Ethereum block headers..
+
+## Abstract
+
+Replace field Difficulty with TotalDifficulty (sum of difficulties since genesis) in the Ethereum block header.
+
+## Motivation
+
+Simplify total difficulty verification in stateless clients.
+
+## Specification
+
+Difficulty field will be replaced with TotalDifficulty (sum of difficulties since genesis) in the Ethereum block header.
+
+## Rationale
+
+This simplifies the selection of the best peer, acceptance of the NewBlock broadcasts and stateless client execution without verification from genesis.
+
+## Backwards Compatibility
+
+Clients will have to understand that headers before the EIP will have Difficulty and not total difficulty field.
+
+## Copyright
+Copyright and related rights waived via
+[CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
Nitpick: These last two lines should probably be one line.

Reply:
I wonder, just copied it from other sample EIP

Reply:
I'm not sure where you might have copied it from, but the template has it one-lined.  :)  https://github.com/ethereum/EIPs/blob/master/eip-template.md

Reply:
In Markdown, this would show up as one line. You need 2 linebreaks to make one linebreak in Github Markdown.